### PR TITLE
GP Internal Core API v1.1 : memory management functions

### PR DESCRIPTION
**Pull request by cedric-chaumont-st-dev**
update panic reasons

Signed-off-by: Cedric Chaumont <cedric.chaumont@st.com>

Note : For testing use optee_test with dev/gp11_meme_mngt branch

### CODE REVIEW

File: lib/libutee/tee_user_mem.h [hunk 1]
```diff
@@ -40,6 +40,6 @@ void tee_user_mem_garbage(void);
  * By default, the hint of a buffer is 0, means the buffer is fill with 0
  * after its allocation
  */
-static const uint32_t DEFAULT_TEE_MALLOC_HINT = 0x0;
+static const uint32_t DEFAULT_TEE_MALLOC_HINT = TEE_MALLOC_FILL_ZERO;
```
Comment (pascal-brand-st-dev) [inline lib/libutee/tee_user_mem.h]: Above comment to be updated

**Comment by pascal-brand-st-dev** [comment]
Once the comment is updated:

    Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jenswi-linaro** [comment]
Ditto
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by cedric-chaumont-st-dev** [comment]
Updated the comment.

**Comment by cedric-chaumont-st-dev** [comment]
Squashed

**Pull request merged**